src/components/BaseComponents/Contributors/Contributor/Contributor.js

```diff
@@ -1,25 +1,54 @@
 import * as React from "react";
 import { useContributor } from "./useContributor";
 import UserPopout from "../../UserPopup";
+import "../../../../styles/components/contributor.scss";
 
 const Contributor = ({ contributor }) => {
     const { user } = useContributor(contributor.author.url);
-    console.log(user);
 
     const [popup, setPopup] = React.useState(false);
+    const popupRef = React.useRef(null);
+    let added = 0,
+        deleted = 0;
+    contributor.weeks.forEach((element) => {
+        added += element.a;
+        deleted += element.d;
+    });
+    React.useEffect(() => {
+        function handleClickOutside(event) {
+            if (popupRef.current && !popupRef.current.contains(event.target)) {
+                setPopup(false);
+            }
+        }
```

Comment:
You are inside of the functional component feel free to use React.useCallback instead.